.github/workflows/check-code.yaml

```diff
@@ -133,3 +133,32 @@ jobs:
             echo "::error::due to bandit violations, return code ${ret}"
             exit 1
           fi
+  check_code_pytest:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 'pypy-3.9'
+          check-latest: true
+      - name: Checkout PyTorch
+        uses: actions/checkout@master
+      - name: Update pip
+        run: pip install --upgrade pip
+      - name: Install pytest
+        run: pip install pytest
+      - name: Checkout plugin current branch
+        uses: actions/checkout@v3
+      - name: Checkout geoips main branch
+        uses: actions/checkout@v3
+        with:
+          repository: NRLMMD-GEOIPS/geoips
+          ref: main
+          token: ${{ secrets.GEOIPS_TOKEN }}
+          path: geoips_dev_utils
+          sparse-checkout: |
+            tests/utils
+      - name: Run code check script pytest
+        run: |
+          pytest -v geoips/xarray_utils/test_dtree.py
```

Comment:
I think this should just run pytest tests/test_pytest
Right @jsolbrig ?

Then it would run all the pytests in any repository?

Reply:
Will address this when implemented - removing from workflow in this PR.